src/controllers/commands/CommandController.cpp

```diff
@@ -538,6 +539,32 @@ void CommandController::initialize(Settings &, Paths &paths)
 
         return "";
     });
+
+    this->registerCommand("/streamlink", [](const QStringList &words,
+                                            ChannelPtr channel) {
+        if (words.size() < 2)
+        {
+            channel->addMessage(makeSystemMessage(
+                "Usage: /streamlink [channel], use \"channel\" to open current "
```

Comment:
Do we have an established syntax for commands and especially command arguments? In [EBNF](https://en.wikipedia.org/wiki/Extended_Backus%E2%80%93Naur_form) (which is also commonly used in man pages), `[ ]` is used to denote that an argument is optional. In this case, the argument is required, and cannot be omitted.

If we have already established this syntax elsewhere, it's fine by me. Otherwise, I suggest changing this to keep in line with common conventions.

Reply:
Sure thing, I went with [] because I changed my mind about whether `"channel"` would be required or not, I guess I will change this to `< >` to keep consistency with EBNF. :+1: